deployment/systemd/setup.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Radioan Service Setup
+
+[Service]
+WorkingDirectory=/opt/radiodan/rde/services/setup
+ExecStart=/usr/local/bin/npm start
+Restart=on-failure
+StandardOutput=syslog
+SyslogIdentifier=rde-debug
+Type=idle
+Environment=PORT=5020
```

Comment:
Is it time to have a policy for port numbering? We currently have `5000`, `5001`, `5002`, `5020`, `5100`, `8080` 😭 .

Reply:
It's nuts, and needs fixing. I have some ideas around setting Environment Variables from external files, which might help with this.

Reply:
To be clear, I'll set it up in another PR that can be merged before this.